Source/WebInspectorUI/UserInterface/Views/FolderTreeElement.js

```diff
@@ -25,11 +25,34 @@
 
 WI.FolderTreeElement = class FolderTreeElement extends WI.GeneralTreeElement
 {
-    constructor(title, representedObject)
+    constructor(title, representedObject, id = null)
     {
         const classNames = [WI.FolderTreeElement.FolderIconStyleClassName];
         const subtitle = null;
         super(classNames, title, subtitle, representedObject, {hasChildren: true});
+
+        if (id) {
+            const settingName = "folder-tree-element-" + id;
```

Comment:
Please use `let`. Web Inspector uses `const` only for truly [constant things](https://trac.webkit.org/wiki/WebInspectorCodingStyleGuide#Tokensspacingindentationsyntax).